src/main/kotlin/net/developerden/devdenbot/commands/info/StackOverflowCommand.kt

```diff
@@ -0,0 +1,32 @@
+package net.developerden.devdenbot.commands.info
+
+import net.developerden.devdenbot.commands.slash.DevDenSlashCommand
+import net.developerden.devdenbot.extensions.await
+import net.developerden.devdenbot.inject.Used
+import net.developerden.devdenbot.serialization.DDBConfig
+import net.developerden.devdenbot.trait.HasConfig
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent
+import net.dv8tion.jda.api.requests.restaction.CommandCreateAction
+import java.net.URL
+
+/**
+ * @author Kyle h
+ */
+
+@Used
+class StackOverflowCommand constructor(
+    override val ddbConfig: DDBConfig,
+) : DevDenSlashCommand(
+    name = "stackoverflow",
+    description = "A helpful way to find answers using stackoverflow"
+), HasConfig {
+    override suspend fun load(action: CommandCreateAction) = Unit
```

Comment:
I'm happy to add this, but I don't think it will work. 
You need to register the command arguments inside `load`